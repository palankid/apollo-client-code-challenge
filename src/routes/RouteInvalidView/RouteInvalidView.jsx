import React from 'react';

const RouteInvalidView = () => {
    return (
        <div className="route-invalid">
            <span className="route-invalid__content">
                Error 404: Page not found
            </span>
        </div>
    );
};

export default RouteInvalidView;
