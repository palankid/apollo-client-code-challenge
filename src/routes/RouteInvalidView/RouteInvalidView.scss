.route-invalid {
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
    position: absolute;

    &__content {
        font-size: 2rem;
        padding: 10px;
        color: #FFFFFF;
        background: linear-gradient(138deg, rgba(255,127,78,1) 0%, rgba(246,76,84,1) 100%);
    }
}
