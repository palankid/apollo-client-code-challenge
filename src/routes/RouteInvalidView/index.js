import RouteInvalidView from './RouteInvalidView';
import './RouteInvalidView.scss';

export default RouteInvalidView;
