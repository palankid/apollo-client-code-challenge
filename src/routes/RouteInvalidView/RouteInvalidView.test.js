import React from 'react';
import { shallow } from 'enzyme';

import RouteInvalidView from './RouteInvalidView';

describe('RouteInvalidView', () => {
    it('should test that component is rendered and not changed unintentionally', () => {
        const wrapper = shallow(
            <RouteInvalidView/>
        );

        expect(wrapper).toMatchSnapshot();
    });
});
