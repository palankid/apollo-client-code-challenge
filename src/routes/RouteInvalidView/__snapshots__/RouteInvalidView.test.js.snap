// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteInvalidView should test that component is rendered and not changed unintentionally 1`] = `
<div
  className="route-invalid"
>
  <span
    className="route-invalid__content"
  >
    Error 404: Page not found
  </span>
</div>
`;
